<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>JASP</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_6" columnstretch="0">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="TabBar" name="tabBar" native="true">
      <property name="styleSheet">
       <string notr="true">QWidget#background {
	border-bottom: 1px solid #919191;
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0.75 #F0F0F0, stop:0.751 #E3E1E2);
}

QPushButton {
	border-top: 1px solid transparent;
	border-left: 1px solid transparent;
	border-bottom: 1px solid #919191 ;
	border-right: 1px solid transparent;

	border-top-left-radius: 0 ;
	border-top-right-radius: 0 ;
	min-width: 36px ;
	padding: 4px 16px;
	margin: 4px 1px 0 1px;
}

QPushButton:checked {
	border-top: 1px solid #919191;
	border-left: 1px solid #919191;
	border-bottom: 1px solid white ;
	border-right: 1px solid #919191;
	background-color: white ;
}

QPushButton#help {
	margin-right: 5px ;
	border-bottom: 1px solid transparent;
}

QPushButton#help:checked {
	/*border-bottom: 1px solid #919191;*/
}

QPushButton#first {
	margin-left: 5px ;
	color: white;
	border-top: 1px solid #349FE3 ;
	border-bottom: 1px solid #919191;
	border-left: none ;
	border-right: none ;
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0.7 #3CAAE7, stop:0.701 #3CAAE7);
}

QPushButton#first:checked {
	border-top: 1px solid #6B767E;
	border-right: 1px solid #3B363E;
	border-bottom: 1px solid #3CAAE7;
	border-left: 1px solid #6B767E;
	background-color: #3CAAE7;
}

QPushButton#Modules{
	color:blue;
	font-size: x-large;
	padding: 0px 0px;
	margin: 0px 0px 0px 0px;
	border-bottom: 1px;
}


QPushButton#Modules:hover:!pressed {
	image:url(:/icons/addition-sign.svg);
}

QPushButton::menu-indicator {
	image: none ;
 }

</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QStackedWidget" name="topLevelWidgets">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="BackStageWidget" name="backStage">
       <property name="minimumSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </widget>
      <widget class="QWidget" name="mainPage">
       <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,1">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>4</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>4</number>
        </property>
        <property name="bottomMargin">
         <number>4</number>
        </property>
        <item>
         <widget class="QStackedWidget" name="ribbon">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">#ribbon {
	/* background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgb(255, 255, 255), stop:0.5 rgb(240, 240, 240)); */
	/*border-bottom: 1.2px solid #C4C4C4 ;*/
}</string>
          </property>
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="RibbonAnalysis" name="ribbonAnalysis">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
          <widget class="RibbonSEM" name="ribbonSEM"/>
          <widget class="RibbonReinforcementLearning" name="ribbonReinforcementLearning"/>
          <widget class="RibbonSummaryStatistics" name="ribbonSummaryStatistics"/>
          <widget class="RibbonMetaAnalysis" name="ribbonMetaAnalysis"/>
          <widget class="RibbonNetworkAnalysis" name="ribbonNetworkAnalysis"/>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="mainWidget" native="true">
          <layout class="QGridLayout" name="gridLayout">
           <property name="leftMargin">
            <number>2</number>
           </property>
           <property name="topMargin">
            <number>2</number>
           </property>
           <property name="rightMargin">
            <number>2</number>
           </property>
           <property name="bottomMargin">
            <number>2</number>
           </property>
           <item row="0" column="0">
            <widget class="QSplitter" name="splitter">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="handleWidth">
              <number>14</number>
             </property>
             <property name="childrenCollapsible">
              <bool>false</bool>
             </property>
             <widget class="QWidget" name="panel_1_Data" native="true">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>0</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout">
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QQuickWidget" name="quickWidget_Data">
                 <property name="resizeMode">
                  <enum>QQuickWidget::SizeRootObjectToView</enum>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
             <widget class="QWidget" name="panel_2_Options" native="true">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <layout class="QGridLayout" name="gridLayout_2">
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item row="1" column="0">
                <widget class="QWidget" name="pageOptions" native="true">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <layout class="QGridLayout" name="pageOptionsLayout">
                  <property name="leftMargin">
                   <number>0</number>
                  </property>
                  <property name="topMargin">
                   <number>0</number>
                  </property>
                  <property name="rightMargin">
                   <number>0</number>
                  </property>
                  <property name="bottomMargin">
                   <number>0</number>
                  </property>
                  <item row="0" column="0">
                   <widget class="QScrollArea" name="scrollArea">
                    <property name="sizePolicy">
                     <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                    <property name="styleSheet">
                     <string notr="true">QWidget#scrollArea {
	border: 1px solid #929292 ;
	/*border-right: 1px solid #929292 ;*/
}</string>
                    </property>
                    <property name="frameShape">
                     <enum>QFrame::NoFrame</enum>
                    </property>
                    <property name="verticalScrollBarPolicy">
                     <enum>Qt::ScrollBarAlwaysOn</enum>
                    </property>
                    <property name="horizontalScrollBarPolicy">
                     <enum>Qt::ScrollBarAlwaysOff</enum>
                    </property>
                    <property name="widgetResizable">
                     <bool>true</bool>
                    </property>
                    <property name="alignment">
                     <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
                    </property>
                    <widget class="QWidget" name="optionsContentArea">
                     <property name="geometry">
                      <rect>
                       <x>0</x>
                       <y>0</y>
                       <width>300</width>
                       <height>714</height>
                      </rect>
                     </property>
                     <property name="sizePolicy">
                      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                       <horstretch>0</horstretch>
                       <verstretch>0</verstretch>
                      </sizepolicy>
                     </property>
                     <layout class="QGridLayout" name="optionsContentAreaLayout">
                      <property name="leftMargin">
                       <number>0</number>
                      </property>
                      <property name="topMargin">
                       <number>0</number>
                      </property>
                      <property name="rightMargin">
                       <number>0</number>
                      </property>
                      <property name="bottomMargin">
                       <number>0</number>
                      </property>
                      <property name="spacing">
                       <number>0</number>
                      </property>
                     </layout>
                    </widget>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item row="0" column="0">
                <widget class="QWidget" name="optionsErrorPanel" native="true">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>24</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">background-color : #FF5555 ;</string>
                 </property>
                 <layout class="QGridLayout" name="gridLayout_7">
                  <property name="leftMargin">
                   <number>12</number>
                  </property>
                  <property name="topMargin">
                   <number>12</number>
                  </property>
                  <property name="rightMargin">
                   <number>12</number>
                  </property>
                  <property name="bottomMargin">
                   <number>12</number>
                  </property>
                  <item row="0" column="0">
                   <widget class="QLabel" name="optionsErrorLabel">
                    <property name="text">
                     <string>Confidence Interval must be between 80 and 99</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </widget>
             <widget class="QFrame" name="panel_3_Results">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>1</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>200</width>
                <height>0</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QWidget#panel_3_Results {
	border: 1px solid #929292 ;
}</string>
              </property>
              <property name="frameShape">
               <enum>QFrame::NoFrame</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QGridLayout" name="gridLayout_3">
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item row="0" column="1">
                <widget class="CustomWebEngineView" name="webViewResults" native="true">
                 <property name="cursor">
                  <cursorShape>ArrowCursor</cursorShape>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
             <widget class="QFrame" name="panel_4_Help">
              <property name="styleSheet">
               <string notr="true">QWidget#panel_4_Help{
	border: 1px solid #919191 ;
}</string>
              </property>
              <layout class="QGridLayout" name="gridLayout_5">
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item row="0" column="0">
                <widget class="CustomWebEngineView" name="webViewHelp" native="true">
                 <property name="acceptDrops">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QQuickWidget</class>
   <extends>QWidget</extends>
   <header>QtQuickWidgets/QQuickWidget</header>
  </customwidget>
  <customwidget>
   <class>RibbonAnalysis</class>
   <extends>QWidget</extends>
   <header>analysis/ribbons/ribbonanalysis.h</header>
   <container>1</container>
   <slots>
    <signal>itemSelected(QString)</signal>
   </slots>
  </customwidget>
  <customwidget>
   <class>RibbonSEM</class>
   <extends>QWidget</extends>
   <header>analysis/ribbons/ribbonsem.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>RibbonSummaryStatistics</class>
   <extends>QWidget</extends>
   <header>analysis/ribbons/ribbonsummarystatistics.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>RibbonReinforcementLearning</class>
   <extends>QWidget</extends>
   <header>analysis/ribbons/ribbonreinforcementlearning.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TabBar</class>
   <extends>QWidget</extends>
   <header>widgets/tabbar.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CustomWebEngineView</class>
   <extends>QWidget</extends>
   <header>widgets/customwebengineview.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BackStageWidget</class>
   <extends>QWidget</extends>
   <header>widgets/backstagewidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>RibbonMetaAnalysis</class>
   <extends>QWidget</extends>
   <header>analysis/ribbons/ribbonmeta_analysis.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>RibbonNetworkAnalysis</class>
   <extends>QWidget</extends>
   <header>analysis/ribbons/ribbonnetworkanalysis.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <slots>
  <slot>tabChanged(int)</slot>
  <slot>dataSetLoaded(DataSet*)</slot>
  <slot>itemSelected(QString)</slot>
 </slots>
</ui>
